providers/edge3/src/airflow/providers/edge3/cli/edge_command.py

```diff
@@ -619,6 +624,98 @@ def stop(args):
         logger.info("Worker has been shut down.")
 
 
+@providers_configuration_loaded
+@provide_session
+def _fetch_edge_hosts_from_db(session: Session = NEW_SESSION) -> list:
+    from airflow.providers.edge3.models.edge_worker import EdgeWorkerModel
+
+    return session.scalars(select(EdgeWorkerModel).order_by(EdgeWorkerModel.worker_name)).all()
+
+
+@cli_utils.action_cli
+@providers_configuration_loaded
+@provide_session
+def list_edge_workers(args, session: Session = NEW_SESSION) -> None:
+    """Query the db to list all registered edge workers."""
+    all_hosts_iter = _fetch_edge_hosts_from_db(session)
+    # Format and print worker info on the screen
+    fields = [
+        "worker_name",
+        "state",
+        "queues",
+        "jobs_active",
+        "jobs_success",
+        "jobs_failed",
+        "jobs_taken",
+        "maintenance_comment",
+    ]
+    all_hosts = [{f: str(host.__getattribute__(f)) for f in fields} for host in all_hosts_iter]
+    # Filter hosts by state
+    if args.state:
+        all_hosts = [host for host in all_hosts if host["state"] in args.state]
+    AirflowConsole().print_as(data=all_hosts, output=args.output)
+
+
+@providers_configuration_loaded
+@provide_session
+def _valid_host(hostname, session: Session = NEW_SESSION) -> bool:
+    """Query the database to check if provided hostname exists."""
+    all_hosts_iter = _fetch_edge_hosts_from_db(session)
+    for host in all_hosts_iter:
+        if hostname == host.worker_name:
+            return True
+    return False
+
+
+@cli_utils.action_cli
+@providers_configuration_loaded
+def put_remote_worker_on_maintainance(args) -> None:
+    """Put remote edge worker on maintenance."""
+    if not _valid_host(args.edge_hostname):
+        raise SystemExit(f"Error: Edge Worker {args.edge_hostname} is unknown!")
+    from airflow.providers.edge3.models.edge_worker import request_maintenance
+
+    maintenance_comment = f"[{datetime.now().strftime('%Y-%m-%d %H:%M')}] - {getuser()} put node into maintenance mode\nComment: {args.comments}"
+    request_maintenance(args.edge_hostname, args.comments)
+    logger.info(f"{args.edge_hostname} has been put on maintenance by {getuser()}!")
```

Comment:
I'd propose w/o exclamation mark. It is no error or warning.
```suggestion
    logger.info(f"{args.edge_hostname} has been put on maintenance by {getuser()}")
```
(and same for the other info logs)